vic/drivers/image/src/vic_init_output.c

```diff
@@ -193,10 +207,167 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining time dimenension in %s", nc->fname);
     }
 
+    // define the netcdf variable time
+    status = nc_def_var(nc->nc_id, "time", NC_INT, 1,
+                        &(nc->time_dimid), &(time_var_id));
+    if (status != NC_NOERR) {
+        log_err("Error defining time variable in %s", nc->fname);
+    }
+    status = nc_put_att_text(nc->nc_id, time_var_id, "standard_name",
+                             strlen("time"), "time");
+    if (status != NC_NOERR) {
+        log_err("Error adding attribute in %s", nc->fname);
+    }
+    if (global_param.output_steps_per_day == 1) {
+        sprintf(strUnit, "days");
+    }
+    else {
+        sprintf(strUnit, "hours");
+    }
+    sprintf(str, "%s since %i-%02i-%02i 00:00:00", strUnit,
+            global_param.startyear, global_param.startmonth,
+            global_param.startday);
```

Comment:
We don't understand why this should give problem. It is in fact the time which is extracted from the global_param and provided by the user.
I expect some problem with analysing software when using this date. An other minor detail is that "0001-01-01 00:00:00" can be confusing for users (users maybe thing it is not a real date).
A date like 1900-01-01 00:00:00 could also be used (it is more intuitive).